src/gsi/m_berror_stats_reg.f90

```diff
@@ -437,8 +440,13 @@ subroutine berror_read_wgt_reg(msig,mlat,corz,corp,hwll,hwllp,vz,rlsig,varq,qopt
         var=upper2lower(varshort)
         if (trim(var) == 'pm25') var = 'pm2_5'
      else 
+        if (laeroana_fv3cmaq .or. berror_fv3cmaq) then
+        read(inerr,iostat=istat) varlong, isig
```

Comment:
A little confusion here.  When laeroana_fv3cmaq=.true., the regular control variables will also be analyzed together,
right?  So, in this case, line 444 will also be applied to regular background files ,in which, "varshort" is supposed to be 
used?   or when laeroana_fv3cmaq .or. berror_fv3cmaq, special berror files would be used, in which varnames for 
regular variables like  t would be written in varlong format?

Reply:
I generated a static B file for RRFS-CMAQ which including both standard meteorological control variables and CMAQ aerosol variables. If a user wants to use meteorological B and an aerosol B from different resource, we will need to future discuss about this.

Reply:
@hongli-wang , Thanks for explaining!   Then, will only berr_fv3cmaq to  be used here be clearer?  Will there be situations in  which berror_fv3cmaq would need to be false, while laerorana_fv3cmq is .true. (or vice versa)?

Reply:
I also changed the berror_fv3camq to berror_fv3_cmaq_regional to exclude the confusion with global fv3-cmaq.